communication/moderators.md

```diff
@@ -1,28 +1,41 @@
 # Community Moderators
 
-The following people are responsible for moderating/administrating Kubernetes communication channels and their home time zone. 
-See our [moderation guidelines](./moderating.md) for policies and recommendations.
+The following people are responsible for moderating/administrating Kubernetes
+communication channels and their home time zone. See our [moderation
+guidelines](./moderating.md) for policies and recommendations.
 
 ## Mailing Lists
 
 ### kubernetes-dev
 
-### Administrators
+#### Administrators (3)
 
 - Sarah Novotny (@sarahnovotny) - PT
 - Brian Grant (@bgrant0607) - PT
```

Comment:
we should ask these folks if they explicitly want to still be included

Reply:
I have no interest in being included. Thanks. :-)

Reply:
i am not able to continue moderating.  thanks for checking.

Reply:
I assume, it's about mailing list ownership, not day-to-day operations (moderation).

At the moment @bgrant0607 @sarahnovotny @parispittman and @spiffxp have "owner" permissions at k-dev.

Reply:
I assume, it's about mailing list ownership, not day-to-day operations (moderation).

At the moment @bgrant0607 @sarahnovotny @parispittman and @spiffxp have "owner" permissions at k-dev.